### ISSUE TITLE
Additional Ceph RBD creation options

### ISSUE THREAD

**Issue by drakonstein** [kubernetes/kubernetes#66029]
/kind feature

**What happened**:
Creating a Ceph RBD PV defaults to --order 22 which is 4MB block sizes that the data stored on a Ceph RBD is stored in the Ceph cluster as.  That is the Ceph default if you don't specify a different block size.

**What you expected to happen**:
It would be nice to be able to specify this in the StorageClass.  The value passed to order is 2^n.  So 22 is 2^2 = 4.  20=1M, 21=2M, 22=4M, 23=8M, 24=16M, 25=32M, and 26=64M.  64MB block sizes is the largest that you can go and 1M is the smallest.

**How to reproduce it (as minimally and precisely as possible)**:
Create a persistent volume with a Ceph RBD and check the RBD info in the ceph cluster for its order.

**Anything else we need to know?**:
What I was able to find in the code for RBD creation is:
https://github.com/kubernetes/kubernetes/blob/2a3144e377182c5d82993d5a47739ece368fe1fc/cluster/juju/layers/kubernetes-master/actions/create-rbd-pv#L122-L123

**Environment**:
- Kubernetes version (use `kubectl version`): v1.9.6

**Comment by wqcheng** [comment]
Hello, my friend, do you have the method to modify the order parameter now?

**Comment by drakonstein** [comment]
@wqcheng No, the options implemented for RBD creation still use a majority of defaults without an option to modify them.  Here is the current line in Master.

https://github.com/kubernetes/kubernetes/blob/master/cluster/juju/layers/kubernetes-master/actions/create-rbd-pv#L147-L148
